deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-service-core.ts

```diff
@@ -169,7 +176,9 @@ export class MigrationServiceCore extends Stack {
                     logGroup: serviceLogGroup
                 })
             },
-            cloudMapOptions: cloudMapOptions
+            cloudMapOptions: cloudMapOptions,
+            minHealthyPercent: 0,
```

Comment:
0 means that none of the containers are required to be up... that doesn't seem correct, shouldn't there be a min threadhold, like 50?  If not could you put a quick inline comment to indicate what the intention of this setting is?

Reply:
Removed in new revision.